packages/components/src/components/table-stateless/readme.md

```diff
@@ -0,0 +1,16 @@
+# kol-table-stateless
```

Comment:
Es fehlt noch die Doku. Kommt die Separat? Dem Nutzer sollte auf jeden Fall klargemacht werden, wann er welche der beiden Tabellen verwenden muss.

Reply:
Erstmal als Draft markieren – analog https://public-ui.github.io/docs/components/split-button (aber Umsetzung als Card)

Doku erfolgt in einem separaten Ticket.

Reply:
Doku doch schon in diesem PR.